tests/test_cli.py

```
@@ -0,0 +1,834 @@
+
+        cli.main()
+        mock_run.assert_called_once()
+
+    @patch("sys.argv", ["biblebot"])
+    @patch("os.path.exists")
+    @patch("builtins.input")
+    @patch("biblebot.cli.generate_config")
+    @patch("biblebot.cli.asyncio.run")
+    @patch("sys.exit")
+    def test_bot_no_config_generate_yes(
+        self, mock_exit, mock_run, mock_generate, mock_input, mock_exists
```

Ruff (0.12.2)

Unused method argument: `mock_exit`

(ARG002)